src/audiomixerboard.cpp

```diff
@@ -862,6 +862,8 @@ void CChannelFader::SetChannelInfos ( const CChannelInfo& cChanInfo )
     plblLabel->setToolTip ( strToolTip );
     plblLabel->setAccessibleName ( strAliasAccessible );
     plblLabel->setAccessibleDescription ( tr ( "Alias" ) );
+    dynamic_cast<QWidget*> ( plblLabel->parent() )
```

Comment:
Just trying to understand why the cast is needed:
- This is in `CChannelFader` which is a `QObject`. Therefore, `parent()` returns a `QObject` as well.
- Doing the dynamic_cast is safe and right to do because we know that `CChannelFader` is only used as a child in `CAudioMixerBoard`, which is a `QWidget`.

Is that correct? If so, would it make sense to add a short code comment about that?

Reply:
Most UI things are widgets.  But if parent (i.e. the form it's stuck to) is only _declared_ as a QObject, you'll get told off by the compiler trying to use methods QObject doesn't have -- _even though_ the QObject you're talking about does have them.  Hence the cast.  So long as it's viable (I've not checked).